### PR TITLE
Add ServicesResetterProcessorConfigurator (closes #170)

### LINKED ISSUE #170: Reset application after each consummed message
As of Symfony 4.4, we can reset application (`$this->getApplication()->reset();`) in a Symfony Command.  

This is usefull to reset Monolog (when finger crossed is used for example), and free resources while waiting for messages.

### PR DESCRIPTION
**Pull request by pvgnd**
Just a work in progress for now to propose an implementation.

Still have tests to write

### CODE REVIEW

**Comment by pvgnd** [comment]
@odolbeau could you have a look on the proposed implementation please ? (still WIP as tests has not been written yet)

File: Processor/ServicesResetter/ServicesResetterProcessor.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        return $this->processor->process($message, $options);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function sleep(array $options)
+    {
+        $this->servicesResetter->reset();
```
Comment (pvgnd) [inline Processor/ServicesResetter/ServicesResetterProcessor.php]: This is convenient to reset services here to ensure that all processors have done their job. But should be better to do it in process() and write a documentation about the processor to explain that it should be the last processor to be executed in the stack. What do you think ?

Comment (odolbeau) [reply Processor/ServicesResetter/ServicesResetterProcessor.php]: The `sleep` method is called only when no message is found in the broker.
If you have a bunch of messages to process, the `servicesResetter` won't be called between all messages.

Comment (odolbeau) [reply Processor/ServicesResetter/ServicesResetterProcessor.php]: See this Doctrine processor to see how it's done: https://github.com/swarrot/swarrot/blob/master/src/Swarrot/Processor/Doctrine/ObjectManagerProcessor.php

**Comment by odolbeau** [comment]
This processor should not be put in the bundle but in https://github.com/swarrot/swarrot instead.
Otherwise, it looks like a promising start, thanks for your work! :)

**Comment by lyrixx** [comment]
Thanks :+1:

**Comment by pvgnd** [comment]
OK, I will move Processor into https://github.com/swarrot/swarrot

File: Processor/ServicesResetter/ServicesResetterProcessor.php (hunk 1 above)
Comment (pvgnd) [reply Processor/ServicesResetter/ServicesResetterProcessor.php]: OK, I will do that

**Comment by pvgnd** [comment]
@odolbeau could you have a look please ?

**Comment by odolbeau** [comment]
Looks great!
Could you squash your commits please?

As swarrot 1.7.0 have been released it will allow us to check test at the same time. :)

**Comment by pvgnd** [comment]
Done

**Comment by odolbeau** [comment]
Perfect, thanks!

**Pull request merged**